lighthouse-core/test/computed/metrics/largest-contentful-paint-all-frames-test.js

```diff
@@ -46,13 +46,16 @@ describe('Metrics: LCP from all frames', () => {
     await expect(resultPromise).rejects.toThrow('NO_LCP_ALL_FRAMES');
   });
 
-  it('should fail if even if main frame LCP is available', async () => {
+  it('should use main frame LCP if no other frames', async () => {
```

Comment:
ah I'm guessing this was the LCP change then? I imagine it might make querying a smidge harder than it was before, but this is consistent with FCP behavior now and I like it 👍

Reply:
Yes, this is the change in https://github.com/GoogleChrome/lighthouse/pull/11874#discussion_r558610694